app/models/user.rb

```diff
@@ -10,6 +11,9 @@ class User < ApplicationRecord
     length: {minimum: Settings.length.min_password}
   has_secure_password
 
+  scope :activated, ->{where activated: true}
+  scope :order_by_id, ->{order(id: :asc)}
```

Comment:
```suggestion
  scope :order_by_id, ->{order :name}
```ai lại đi sắp xếp theo id trùi
asc là mặc định lượt bỏ dc